daemons/controld/controld_te_utils.c

```diff
@@ -229,7 +229,11 @@ tengine_stonith_notify(stonith_t * st, stonith_event_t * st_event)
          */
 
 #ifdef RB_HALT_SYSTEM
-        reboot(RB_HALT_SYSTEM);
+        reboot(RB_HALT_SYSTEM
+#ifdef ON_SOLARIS
+           , ""
+#endif
+        );
```

Comment:
This is harder to read ... how about a macro/inline wrapper in include/crm/common/internal.h?

Reply:
Considered that, but there are just 2 occurrences, so not sure it's
worth it.  What I checked though, is whether `gnulib` defines a compat
wrapper to be used right away (nope).

Reply:
Yes, I think the readability is worth it.

Reply:
Ok, but `include/portability.h` looks like a better fit.